docs/pages/fase_02/iniciativasExtras/estimativas.md

```diff
@@ -0,0 +1,201 @@
+| ------ | :--------: | :----------------------: | :----------------: | :-------------------: |
+| 1.0    | 14/02/2022 |   Criação do Documento   | Giulia Lobo Barros | Murilo Gomes de Souza |
+| 1.1    | 15/02/2022 | Atualização do Documento | Giulia Lobo Barros |      Pedro Lima       |
+
+<!-- NÃO ESQUECER DE ADICIONAR AO "/_sidebar.md" -->
+
+## Introdução
+
+Uma das ações mais importantes para a execução de um projeto é fazer estimativas que ajudam na tomada de decisões de líderes e liderados. Essas estimativas podem ajudar a evitar grandes riscos, impedimentos e problemas no futuro e por isso são de grande valia.
+
+Tendo isso em consideração, o grupo decidiu que as estimativas que mais fariam sentido para a equipe seriam as de **esforço**, **tempo** e **custo** e que elas ajudariam a mapear um projeto realista, não muito complexo, dentro do prazo acordado com a professora e de baixo custo.
+
+Por esse motivo, o método que mais se encaixou com as propostas e acabou por ser o escolhido para auxiliar o projeto, foi o **Construtive Cost Model**, mais conhecido como **COCOMO**.
+
+Esse método de estimativa do tempo e esforço de desenvolvimento de um software, que foi criado por Barry Boehm, por ser baseado no esforço de linhas de código de um software, auxilia na criação de deduções, constantes e atributos para cada situação e, por esse motivo, funciona muito bem para projetos de software em geral.
+
+## Metodologia
+
+Tendo o método **COCOMO** apresentado, podemos mostrar como será utilizado pelo grupo e detalhá-lo da seguinte forma:
+
+* *Primeiramente, é importante destacar que o COCOMO é dividido em três implementações, sendo elas:*
+    * **COCOMO Básico (Basic)**: Que computa o esforço e o custo de desenvolvimento considerando uma estimativa do tamanho do programa;
+    * **COCOMO Intermediário (Intermediate)**: Que computa o esforço e o custo de desenvolvimento considerando uma estimativa do tamanho do programa e um conjunto de direcionadores de custo (avaliações subjetivas do produto, do hardware, do pessoal e dos atributos do projeto);
+    * **COCOMO Detalhado (Detailed)**: Que, além das características do COCOMO intermediário, inclui uma avaliação do impacto dos direcionadores de custo sobre cada etapa do desenvolvimento.
+* *Posteriomente, é necessário saber que ele também pode ser aplicado em três classes de projeto, que são:*
+    * **Modo Orgânico**: projetos simples, relativamente pequenos, com conjuntos de requisitos não tão rígidos, com equipes pequenas e experientes;
+    * **Modo Semidestacado**: projetos intermediários (em tamanho e complexidade), com alguns requisitos rígidos e outros não tão rígidos, com níveis mistos de experiência nas equipes;
+    * **Modo Embutido**: projetos com conjunto rígido de restrições operacionais, tanto de hardware, quanto de software.
+
+    *E que essas possuem coeficientes, representados pela tabela a seguir:*

```

Comment:
Fazer uma correção nessa linha. Essa frase deve ser em itálico e deve estar alinhada com o texto em itálico mais acima